tests/core/test_library_files.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from geth.exceptions import (
+    GethError,
+)
+
+PY_GETH_PATH = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "..", "..", "geth"
+)
+DEFAULT_EXCEPTIONS = (
+    AssertionError,
+    AttributeError,
+    FileNotFoundError,
+    GethError,
+    KeyError,
+    NotImplementedError,
+    OSError,
+    TypeError,
+    ValueError,
+)
+
+
+def test_no_default_exceptions_are_raised_within_py_geth():
+    for root, _dirs, files in os.walk(PY_GETH_PATH):
+        for file in files:
+            if file.endswith(".py"):
+                file_path = os.path.join(root, file)
```

Comment:
nit: I don't think the file path is quite right: 
<img width="1123" alt="image" src="https://github.com/ethereum/py-geth/assets/6540608/9667b51b-549f-490c-a587-20e63212aa1e">

Reply:
Hrm. Not sure how you're getting that error - the error on line 42 of `accounts.py` is already a `PyGethValueError`. If I change it to just a `ValueError` and run the test again, it catches it and gives the error message you have.